webapp/publisher/snaps/views.py

```diff
@@ -806,6 +806,45 @@ def post_register_name():
     return flask.redirect(flask.url_for("account.get_account"))
 
 
+@publisher_snaps.route("/register-name-dispute")
```

Comment:
Do we want `/register-name-dispute` or something like `/register-name/dispute` or other?

@therealjuan @Lukewh any preference on that?

Reply:
I think this is OK for now